app/assets/javascripts/visualizations/highvis/bar.coffee

```diff
@@ -103,6 +103,30 @@ $ ->
             [fieldTitle(data.fields[fid]), groupedData[fid][gid]]
 
           @chart.addSeries options, false
+
+          # Draw error bars if that analysis type is selected
+          if @configs.analysisType == @ANALYSISTYPE_MEAN_ERROR
+            errors =
+              type: 'errorbar'
+              name: "Error for #{data.groups[gid]}" or data.noField()
+
+            errors.data = for fid in data.normalFields when fid in fieldSelection
+              barData = data.selector fid, gid
+              if barData.length == 1
+                []
+              else
+                mean = groupedData[fid][gid]
+                innerStd = barData.map((x) -> (x - mean) ** 2).reduce((x, y) -> x + y)
+                stdDev = Math.sqrt((1 / (barData.length - 1)) * innerStd)
+                if globals.configs.logY != true
+                  [mean - stdDev, mean + stdDev]
+                else if mean > 0
+                  [mean, mean + stdDev]
+                else
+                  []
```

Comment:
Means that are negative on a logarithmic y axis are useless enough that they break the vis when you render the error bars over them.

`[mean, mean + stdDev]` renders an error bar from `mean` to `mean + stdDev`.  If `mean <= 0`, this throws an exception because some part of the error bar exists at or below zero.  Negative bars just don't show up on log y-axis either, so it'd be weird if I drew an error bar over a bar you can't see.